common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketch.java

```diff
@@ -67,13 +78,11 @@
       this.versionNumber = versionNumber;
     }
 
-    public int getVersionNumber() {
+    int getVersionNumber() {
       return versionNumber;
     }
   }
 
-  public abstract Version version();
```

Comment:
cc @liancheng , I removed this as the design doc says users should not care about the version being used.